perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -107,7 +114,10 @@
 
 UPLOAD_THROUGHPUT_VIA_CLI = 'upload throughput via cli Mbps'
 DOWNLOAD_THROUGHPUT_VIA_CLI = 'download throughput via cli Mbps'
+
 CLI_TEST_ITERATION_COUNT = 100
+LARGE_CLI_TEST_ITERATION_COUNT = 20
+
 CLI_TEST_FAILURE_TOLERANCE = 0.05
 # Azure does not parallelize operations in its CLI tools. We have to
```

Comment:
Someone mentioned a while ago Azure supports parallelize operations nowadays. Maybe add that in next PR?